content/guides/references/experiments.md

```diff
@@ -39,11 +39,51 @@ creating `e2e` and `component` objects inside your Cypress configuration.
 These experiments are available to be specified inside the `e2e` configuration
 object:
 
-| Option                           | Default | Description                                                                                               |
-| -------------------------------- | ------- | --------------------------------------------------------------------------------------------------------- |
-| `experimentalStudio`             | `false` | Generate and save commands directly to your test suite by interacting with your app as an end user would. |
-| `experimentalRunAllSpecs`        | `false` | Enables the "Run All Specs" UI feature, allowing the execution of multiple specs sequentially.            |
-| `experimentalOriginDependencies` | `false` | Enables support for `require`/`import` within `cy.origin`.                                                |
+| Option                            | Default | Description                                                                                               |
+| --------------------------------- | ------- | --------------------------------------------------------------------------------------------------------- |
+| `experimentalStudio`              | `false` | Generate and save commands directly to your test suite by interacting with your app as an end user would. |
+| `experimentalRunAllSpecs`         | `false` | Enables the "Run All Specs" UI feature, allowing the execution of multiple specs sequentially.            |
+| `experimentalOriginDependencies`  | `false` | Enables support for `require`/`import` within `cy.origin`.                                                |
+| `experimentalSkipDomainInjection` | `null`  | Disables injecting `document.domain` into `text/html` pages.                                              |
+
+#### Experimental Skip Domain Injection
+
+Under the hood, Cypress
+[injects document.domain](/guides/guides/web-security#Examples-of-what-Cypress-does-under-the-hood)
```

Comment:
I just read the Examples-of-what-Cypress-does-under-the-hood section. It doesn't state WHY we set document.domain. This is primarily to allow sub-domain navigation correct? maybe call that out here?

Reply:
We have a spot for this in the cross origin testing guide. I think I can move the workaround up there and it will fit fairly well